webrtc.html

```diff
@@ -1620,8 +1623,80 @@ <h4>Set the RTCSessionDescription</h4>
+                        <p>If <var>connection</var>.<a>[[\PendingLocalDescription]]</a>
+                        is not <code>null</code>, then set <var>description</var> to
+                        <var>connection</var>.<a>[[\CurrentLocalDescription]]</a>,
+                        and set <var>remote</var> to <code>false</code>. Otherwise,
+                        set <var>description</var> to
+                        <var>connection</var>.<a>[[\CurrentRemoteDescription]]</a>,
+                        and set <var>remote</var> to <code>true</code>.
+                        <div class="note">
+                          <var>description</var> is set here for the purpose of
+                          ensuring the remaining algorithm fires events to reflect
+                          the rolled back state. The special case of rolling back
+                          an initial offer is handled immediately below. Only
+                          offers can be rolled back.
+                        </div>
+                      </li>
+                      <li>
+                        <p>If <var>description</var> is now <code>null</code> then
+                        for each <code><a>RTCRtpTransceiver</a></code>,
+                        <var>transceiver</var>, that is associated with a
+                        <a>media description</a> in
+                        <var>connection</var>.<a>[[\PendingRemoteDescription]]</a>,
+                        run the following sub steps:</p>
+                        <ol>
+                          <li>
+                            <p><a>Set the associated remote streams</a> given
+                            <var>transceiver</var>.<a>[[\Receiver]]</a>,
+                            an empty list, another empty list, and
+                            <var>removeList</var>.</p>
+                          </li>
+                          <li data-tests="RTCRtpTransceiver.https.html">
+                            <p><a>process the removal of a remote track</a>
+                            for the <a>media description</a>, given
+                            <var>transceiver</var> and
+                            <var>muteTracks</var>.</p>
+                          </li>
+                        </ol>
+                      </li>
+                      <li>
+                        <p>Set <var>connection</var>.<a>[[\PendingLocalDescription]]</a>
+                        and <var>connection</var>.<a>[[\PendingRemoteDescription]]</a>
+                        to <code>null</code>, and set <a>signaling state</a> to
+                        <code>"stable"</code>.</p>
+                      </li>
+                    </ol>
+                  </li>
+                  <li>
+                    <p>If <var>remote</var> is <code>false</code>, then run the
+                    following steps:</p>
```

Comment:
Now that rolling back the initial offer makes it possible for description to be null, I think this needs to say "If remote is false _and description is not null_, then run the following steps:

Reply:
Hang on a minute, from this step and onward, why are we running different steps based on if remote is true or false? Applying a rollback should have the same effect, regardless if we do SLD(rollback) or SRD(rollback).

Reply:
> I think this needs to say "If remote is false and description is not null

The sole step is "Run the following steps for each media description in description", so that might be implied, but yes I'll clarify this (and save an indent)

> why are we running different steps based on if remote is true or false? Applying a rollback should have the same effect, 

It has. See line 1652: *"If connection.[[PendingLocalDescription]] is not null, then set description to connection.[[CurrentLocalDescription]], and set remote to false. Otherwise, set description to connection.[[CurrentRemoteDescription]], and set remote to true. "*